spec/datetimeformat.html

```diff
@@ -111,6 +93,23 @@ <h1>InitializeDateTimeFormat ( _dateTimeFormat_, _locales_, _options_ )</h1>
         1. Else,
           1. Assert: _hour12_ is *undefined*.
         1. Set _dateTimeFormat_.[[HourCycle]] to _hc_.
+        1. Let _timeZone_ be ? Get(_options_, *"timeZone"*).
+        1. If _timeZone_ is *undefined*, then
+          1. Let _timeZone_ be DefaultTimeZone().
+        1. Else,
+          1. Let _timeZone_ be ? ToString(_timeZone_).
+          1. If the result of IsValidTimeZoneName(_timeZone_) is *false*, then
+            1. Throw a *RangeError* exception.
+          1. Let _timeZone_ be CanonicalizeTimeZoneName(_timeZone_).
+        1. Set _dateTimeFormat_.[[TimeZone]] to _timeZone_.
+        1. Let _formatOptions_ be a new Record.
+        1. For each row of <emu-xref href="#table-datetimeformat-components"></emu-xref>, except the header row, in table order, do
+          1. Let _prop_ be the name given in the Property column of the row.
+          1. If _prop_ is *"fractionalSecondDigits"*, then
+            1. Let _value_ be ? GetNumberOption(_options_, *"fractionalSecondDigits"*, 1, 3, *undefined*).
+          1. Else,
+            1. Let _value_ be ? GetOption(_options_, _prop_, *"string"*, &laquo; the strings given in the Values column of the row &raquo;, *undefined*).
+          1. Set _formatOptions_.[[&lt;_prop_&gt;]] to _value_.
         1. Set _formatOptions_.[[hourCycle]] to _hc_.
```

Comment:
Minor suggestion to avoid post hoc alias reuse:
```suggestion
        1. Set _formatOptions_.[[hourCycle]] to _dateTimeFormat_.[[HourCycle]].
```